desktop/apps/galleries_institutions/components/primary_carousel/template.jade

```diff
@@ -1,52 +1,54 @@
 - var length = profiles.length
 
-if length > 1
-  .gpc-bumpers.js-gpc-navigation
-    .gpc-bumper-prev.gi-carousel-arrow-prev.js-gpc-prev: i.icon-chevron-left
-    .gpc-bumber-next.gi-carousel-arrow-next.js-gpc-next: i.icon-chevron-right
+if length > 0
```

Comment:
Only render the carousel at all if we have profiles to show. This one line is the main change, everything else below is just whitespace changes from adding that `if`